kibana-reports/public/components/context_menu/context_menu.js

```diff
@@ -61,7 +68,7 @@ const replaceQueryURL = () => {
   return url;
 };
 
-const generateInContextReport = (timeRanges, queryUrl, fileFormat) => {
+const generateInContextReport = (timeRanges, queryUrl, fileFormat, rest = {}) => {
```

Comment:
What's the function of rest?

Reply:
This is to add `saved_search_id` field in data report (csv) requests. Line 181 uses it.